### PR TITLE
short array syntax, mirror formatting

### PR DESCRIPTION
**Pull request by torvista**
wrapped long lines, easier to see in code and with comparison, not easy to see here.
No functional changes.

### CODE REVIEW

File: admin/includes/modules/document_product/collect_info_metatags.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
                                                FROM " . TABLE_META_TAGS_PRODUCTS_DESCRIPTION . "
                                                WHERE products_id = " . (int)$_GET['pID']);
   if ($check_meta_tags_description->RecordCount() <= 0) {
-    $product = $db->Execute("SELECT pd.products_name, p.products_model, p.products_price_sorter,
+      $product = $db->Execute("SELECT pd.products_name, p.products_model, p.products_price_sorter,
```
Comment (scottcwilson) [inline admin/includes/modules/document_product/collect_info_metatags.php:34]: Please remove these extra spaces in all three files where you have made the same change.

**Review by scottcwilson — CHANGES_REQUESTED** [review]

File: admin/includes/modules/document_product/collect_info_metatags.php (hunk 1 above)
Comment (torvista) [reply admin/includes/modules/document_product/collect_info_metatags.php:34]: PSR-12 says 4 spaces for ident.
And tangentially:https://github.com/zencart/zencart/pull/5073

Comment (scottcwilson) [reply admin/includes/modules/document_product/collect_info_metatags.php:34]: Yes but the rest of the file is 2.  Please remove and I will merge.

Comment (torvista) [reply admin/includes/modules/document_product/collect_info_metatags.php]: all two.

**Comment by scottcwilson** [comment]
@torvista Please fix so we can get this merged.  Thanks.

**Comment by scottcwilson** [comment]
Please remove the white space changes lines 34 and 43. Don’t need these changes.

**Comment by torvista** [comment]
ok

**Comment by scottcwilson** [comment]
Thanks.
Until we release 1.5.8b, please hold on changes like this (formatting/syntax updates).

**Pull request merged**